src/cms/templates/events/event_form.html

```diff
@@ -227,32 +235,36 @@ <h1 class="heading">
                         </div>
                     </div>
                     <div>
-                        <label for="{{ event_form.location.id_for_label }}">{{ event_form.location.label }}</label>
-                        {% trans 'Name of event location' as poi_title_placeholder %}
-                        <div class="relative my-2">
-                            <input id="poi-query-input" type="search" autocomplete="off" class="pr-8 appearance-none block w-full bg-white text-gray-800 placeholder-gray-800 border border-gray-400 rounded py-3 px-4 leading-tight focus:outline-none focus:bg-white focus:placeholder-gray-600"{% if event_form.disabled %} disabled{% endif %} placeholder="{% if poi %}{{ poi|poi_translation_title:current_language }}{% else %}{{ poi_title_placeholder }}{% endif %}" data-url="{% url 'search_poi_ajax' %}" data-region-slug="{{ region.slug }}" data-default-placeholder="{{ poi_title_placeholder }}">
-                            <div class="absolute inset-y-0 right-0 flex items-center px-2 text-gray-800">
-                                <button id="poi-remove" title="{% trans 'Remove location from event' %}"{% if event_form.disabled %} disabled{% endif %}>
-                                    <i data-feather="trash-2" class="h-5 w-5"></i>
-                                </button>
+                        {% render_field event_form.has_not_location %}
+                        {{ event_form.has_not_location.label_tag }}
```

Comment:
Interesting, I didn't know about this tag.
I think we should use this for all forms in the future :smile: